### PR TITLE
Add disclaimer fields to financial_release

### PR DESCRIPTION
**Pull request by mathildathompson**

### CODE REVIEW

File: formats/financial_release/frontend/examples/financial_release.json [hunk 1]
```diff
@@ -7,8 +7,10 @@
   "locale": "en",
   "public_updated_at": "2015-12-01T00:00:00.000Z",
   "details": {
+    "header_legal_disclaimer": "header legal disclaimer example text",
```
Comment (boffbowsh) [inline formats/financial_release/frontend/examples/financial_release.json]: Might be good to make these examples HTML in order to illustrate the point better.

Comment (mathildathompson) [inline formats/financial_release/frontend/examples/financial_release.json]: @boffbowsh :+1: I have changed to html

File: formats/financial_release/frontend/examples/financial_release.json [hunk 2]
```diff
@@ -7,8 +7,10 @@
   "locale": "en",
   "public_updated_at": "2015-12-01T00:00:00.000Z",
   "details": {
+    "header_legal_disclaimer": "<p><small>Dummy content for header legal disclaimer<p></small>",
```
Comment (boffbowsh) [inline formats/financial_release/frontend/examples/financial_release.json]: This isn't valid HTML, the `p` and `small` are nested inside each other

Comment (mathildathompson) [inline formats/financial_release/frontend/examples/financial_release.json]: :-( third time lucky........

**Pull request merged**